tests/local.rs

```diff
@@ -0,0 +1,264 @@
+#[macro_use]
+extern crate serial_test_derive;
+extern crate fantoccini;
+extern crate futures_util;
+
+use fantoccini::{error, Client};
+use warp::Filter;
+use std::path::PathBuf;
+
+macro_rules! tester {
```

Comment:
It'd be good if we could share most of this code between the remote and local testing code. Maybe in `tests/common.rs`?